docs/release-notes/.FSharp.Compiler.Service/9.0.300.md

```diff
@@ -16,6 +16,7 @@
 * Add support for C# `Experimental` attribute. ([PR #18253](https://github.com/dotnet/fsharp/pull/18253))
 * Nullness warnings are issued for signature<>implementation conformance ([PR #18186](https://github.com/dotnet/fsharp/pull/18186))
 * Symbols: Add FSharpAssembly.IsFSharp ([PR #18290](https://github.com/dotnet/fsharp/pull/18290))
+* Type checker: don't suppress errors while checking expressions ([PR #18311](https://github.com/dotnet/fsharp/pull/18311))
```

Comment:
I think we will need better real-life IDE tests and what it does when typing incomplete snippets of code.
I assume there was a reason to suppress typecheck errors in case of parser errors, to prevent a wall of semantic errors jumping around when typing.

I admit the reasons behind it might have changed, but it would be safer if we could either test this more while typing.